ios/FluentUI/Core/Theme/FluentTheme.swift

```diff
@@ -24,10 +24,8 @@ import SwiftUI
     /// - Parameters:
     ///   - tokens: A custom set of tokens.
     ///   - control: The control to use custom tokens on.
-    public func register<T: TokenizedControl>(tokens: T.TokenType, for control: T) {
-        tokens.globalTokens = globalTokens
-        tokens.aliasTokens = aliasTokens
-        controlTokens[tokenKey(control.tokenKeyComponents)] = tokens
+    public func register<T: TokenizedControl>(controlType: T.Type, tokens: @escaping (T) -> T.TokenType) {
```

Comment:
This doesn't seem to be used anymore (probably related to the comment below on overriding control tokens per theme in a future PR).

I think it might be worth removing it and reintroducing it in the next PR when you get the control tokens customization per theme figured out.

Reply:
Interesting point! So this function was originally designed to be used exclusively by clients; I repurposed it at one point to cover internal scenarios as well, but that was never its intended goal.

That said, this API is hard to verify without the testing work I've got planned for this week or next. I've done light local verification, so I'm confident that it works, but yeah it's unfortunate that there's no proper tests yet.

Would you be comfortable with me leaving it for the moment, in order to keep my future testability PR cleaner?